controls/os_spec.rb

```diff
@@ -236,3 +238,23 @@
     its(:group) { should match(/^root|syslog$/) }
   end
 end
+
+control 'os-12' do
+  impact 1.0
+  title 'Detect vulnerabilities in the cpu-vulnerability-directory'
+  desc 'Check for known cpu vulnerabilities described here: https://www.kernel.org/doc/html/v5.6/admin-guide/hw-vuln/index.html'
+
+  if file(cpuvulndir).exist?
+    describe file(cpuvulndir) do
+      it { should be_directory }
+    end
+
+    loaded_files = command('find ' + cpuvulndir + ' -type f -maxdepth 1').stdout.split(/\n/).map(&:strip).find_all { |vulnfiles| !vulnfiles.empty? }
+
+    loaded_files.each do |vulnfile|
+      describe file(vulnfile) do
+        its('content') { should_not match 'vulnerable' }
```

Comment:
According to the docs it's `Vulnerable` with an uppercase V. Does the regex support this or do we need to change it?

Reply:
I have tested and is lower case. We can keep both just in case. What do you think?

```rb
     ×  File /sys/devices/system/cpu/vulnerabilities/itlb_multihit content is expected not to match "vulnerable"
     expected "Processor vulnerable\n" not to match "vulnerable".
     Diff:
     @@ -1 +1 @@
     -vulnerable
     +Processor vulnerable
```

Reply:
Supporting both cases sounds like a good idea. We then should probably use `should_not cmp`: https://docs.chef.io/inspec/matchers/#cmp

Reply:
@rndmh3ro `should_not cmp` doesn't fit to this context. For file content it works as shown. (I have taken other references. Find similar code in this file. Thank you!